packages/modules/common/b32.py

```diff
@@ -1,29 +1,48 @@
 #!/usr/bin/env python3
-from typing import List, Tuple
+from typing import Callable, List, Tuple
 
 from modules.common import modbus
+from modules.common.fault_state import FaultState
 from modules.common.modbus import ModbusDataType
 
 
+def exceptions_to_fault_state(delegate: Callable):
```

Comment:
Ich glaube nicht, dass die Methode in jedem einzelnen Modul Sinn macht. Das kann man doch auslagern und dann nur importieren? Würde rein inhaltlich z. B. gut in `modules.common.fault_state` passen.